pkg/templates/images/images.go

```diff
@@ -165,9 +165,9 @@ func FindResource(name string) (Resource, error) {
 
 func baseResources() map[Resource]map[string]string {
 	return map[Resource]map[string]string{
-		CalicoCNI:         {"*": "docker.io/calico/cni:v3.22.0"},
-		CalicoController:  {"*": "docker.io/calico/kube-controllers:v3.22.0"},
-		CalicoNode:        {"*": "docker.io/calico/node:v3.22.0"},
+		CalicoCNI:         {"*": "docker.io/calico/cni:v3.22.2"},
+		CalicoController:  {"*": "docker.io/calico/kube-controllers:v3.22.2"},
+		CalicoNode:        {"*": "docker.io/calico/node:v3.22.2"},
```

Comment:
Not really relevant to this PR but maybe we should switch to `quay.io` for these images. They maintain the same images on docker hub and quay.io 

https://github.com/projectcalico/calico/blob/master/.semaphore/push-images/cni-plugin.yml#L43

Reply:
@ahmedwaleedmalik send patch!